src/RequestContextProvider.php

```diff
@@ -8,23 +8,42 @@
 
 class RequestContextProvider implements ContextProviderInterface
 {
-    /** @var Request */
+    /**
+     * The current request.
+     *
+     * @var \Illuminate\Http\Request|null
+     */
     private $currentRequest;
 
-    /** @var VarCloner  */
+    /**
+     * The variable cloner.
+     *
+     * @var \Symfony\Component\VarDumper\Cloner\VarCloner
+     */
     private $cloner;
 
+    /**
+     * RequestContextProvider constructor.
+     *
+     * @param  \Illuminate\Http\Request|null  $currentRequest
+     * @return void
+     */
     public function __construct(Request $currentRequest = null)
     {
         $this->currentRequest = $currentRequest;
-        $this->cloner = new VarCloner();
+        $this->cloner = new VarCloner;
         $this->cloner->setMaxItems(0);
     }
 
+    /**
+     * Get the context.
+     *
+     * @return array|null
+     */
     public function getContext(): ?array
     {
-        if (null === $this->currentRequest) {
-            return null;
+        if ($this->currentRequest === null) {
+            return;
```

Comment:
When using a nullable return type, the return MUST be explicit with either the typehinted return type or null. Otherwise you'll get an error. Confirmed on PHP 7.2.9.

Reply:
@devcircus good catch. I haven't tagged a new release yet. I'll fix this.